src/foraging_gui/Foraging.py

```diff
@@ -1741,19 +1741,19 @@ def _GetSaveFolder(self,CTrainingFolder=1,CHarpFolder=1,CVideoFolder=1,CPhotomet
 
     def _GetSaveFileName(self):
         '''Get the name of the save file. This is an old data structure and has been deprecated.'''
-        SaveFileMat = os.path.join(self.default_saveFolder, self.Tower.currentText(),self.ID.text(), f'{self.ID.text()}_{date.today()}.mat')
-        SaveFileJson= os.path.join(self.default_saveFolder, self.Tower.currentText(),self.ID.text(), f'{self.ID.text()}_{date.today()}.json')
-        SaveFileParJson= os.path.join(self.default_saveFolder, self.Tower.currentText(),self.ID.text(), f'{self.ID.text()}_{date.today()}_par.json')
+        SaveFileMat = os.path.join(self.default_saveFolder, self.ID.text(), f'{self.ID.text()}_{date.today()}.mat')
```

Comment:
I suggest we keep the original folder structure to contain the box so we can easily identify which box the session came from without opening the json file.